docs/supported-technologies.asciidoc

```diff
@@ -87,16 +87,13 @@ These are the frameworks that we officially support:
 
 [options="header"]
 |=======================================================================
```

Comment:
REVIEW NOTE: aThe changes in this file are all either:
- Formatting changes to be more consistent (which helped the parsing of this file being done in "bitrot.js"), or
- Replacing some weird unicode-space characters of unknown origin with simple ASCII space.